Report DNS resolution failure

renajohn opened the pull request:
When an INET_E_RESOURCE_NOT_FOUND error is reported
on navigation, it means the DNS resolution failed or a connection failed. There is no way to distinguish those two cases. This PR is adding a comment in the code.

ocr commented on agent/browser/ie/wptbho/wpt.cc:
```diff
@@ -343,6 +342,10 @@ void Wpt::OnNavigateError(DWORD error) {
     case INET_E_CODE_INSTALL_BLOCKED_IMMERSIVE:
       code = 12999L;
       break;
+    // DNS resolution failed. Connection failure is reported with INET_E_CONNOT_CONECT
```
Typo in comment: CONNOT -> CANNOT

kart replied:
Why is this comment placed above INET_E_RESOURCE_NOT_FOUND?

renajohn replied:
You prefer it below? I'll fix the type.

ocr commented:
Other than the typo, 👍

renajohn commented:
I've addressed all comments but I have second thought on merging this PR. INET_E_RESOURCE_NOT_FOUND can also be a connection error, like a proxy blocking the connection.

ocr commented:
👍
